dotCMS/src/test/java/com/liferay/util/FileUtilTest.java

```diff
@@ -2,19 +2,118 @@
 
 import java.io.File;
 import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
 import java.nio.file.Files;
 import java.util.Calendar;
 import java.util.Date;
 import java.util.List;
 
+import com.dotcms.util.CloseUtils;
+import org.junit.Assert;
 import org.junit.Test;
 
 import com.google.common.collect.ImmutableList;
 import static org.junit.Assert.assertEquals;
 
 public class FileUtilTest {
 
-  @Test
+    @Test
+    public void test_uncompress_outputstream()  throws Exception {
+
+        final File file = com.dotmarketing.util.FileUtil.createTemporalFile("nocompre_");
+        final OutputStream out = FileUtil.createOutputStream(file);
+
+        final String  string = "testuncompress";
+        final byte [] bytes  = string.getBytes();
+        out.write(bytes);
+        CloseUtils.closeQuietly(out);
+
+        final InputStream in   = FileUtil.createInputStream(file);
+        final byte [] buff     = new byte[bytes.length];
+        in.read(buff);
+        CloseUtils.closeQuietly(in);
+
+        Assert.assertEquals("The file recovery should be the file saved", new String(buff), string);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "The file recovery should be the file saved" appears 5 times in this file; the first occurrence is on line 37](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4960665)